core/src/impl/Kokkos_Atomic_Memory_Order.hpp

```diff
@@ -51,8 +51,19 @@ struct memory_order_relaxed_t {
 #endif
   static constexpr auto std_constant = std::memory_order_relaxed;
 };
+
+// FIXME_OPENMPTARGET - The `declare target` is needed for the Intel GPUs with
+// the OpenMPTarget backend
+#if defined(KOKKOS_ENABLE_OPENMPTARGET) && defined(KOKKOS_ARCH_INTEL_PVC)
+#pragma omp declare target
+#endif
+
 constexpr memory_order_relaxed_t memory_order_relaxed = {};
```

Comment:
Did you try declaring it `inline` instead?

Reply:
didn't work on the ALL one, so ...

Reply:
sorry did not notice it does not work for `ALL`

Reply:
@masterleinad Did you try inline? I don't have access to that GPU.